exercise_name4.rb

```diff
@@ -0,0 +1,21 @@
+more = true
+while more==true
+
+def my_method(word)
```

Comment:
I think you can find a better name than `my_method`! 

An approach for that is to try to code by intention. How would you communicate your intention through the code? would it be comprehensible by someone that doesn't know anything about coding?

For example:
``` ruby
my_method("verb")
```
could be something like
``` ruby
ask_user_a("verb")
```

Reply:
Agreed, I'll change it to something more explicit!